I-D_in_xml/draft-ietf-ccamp-optical-impairment-topology-yang-16.txt

```diff
@@ -0,0 +1,8232 @@
+      |                          |     |    |         \--o    | LTP 3|
+      |                          |     |    o----\       | 3  o------o->
+      |                          o      \___|     \------o    |   |  |
+      |                          |                       |___/    |  |
+      |                          |                            DEG3|  |
+      |                          +--------------------------------+  |
+      +--------------------------------------------------------------+
+
+
+
+
+Beller, et al.           Expires 9 January 2025               [Page 106]
+
+Internet-Draft    Opt. Impairment-Aware Topo YANG Model        July 2024
+
+
+       Figure 28: Protected TTP with Full Connectivity between ROADM
+                   Add-Drop Ports and ROADM Degree Ports
+
+      +--------------------------------------------------------------+
+      |                                       ROADM                  |
+      |                          +--------------------------------+  |
+      |  Local OT     Splitter   |                        ___     |  |
+      |  +-------+   +-------+  AD1       ___            |   \  Line |
+      |  |    TTP|   |    ---o-->o--     /   |       /---o    | LTP 1|
+      |  |  +----|   |   /   |   |  \   |    o------/    | 1  o------o->
+    --o->|  | Tx o-->o--o  5 |   |   ---o  4 |         /-o    |   |  |
+      |  |  +----|   |   \   |  AD2     |    o-\      /  |___/    |  |
+    <-o--|  | Rx o   |    ---o-->o--     \___|  \    /        DEG1|  |
+      |  |  +----|   +-------+   |  \     ___    \   |    ___     |  |
+      |  |       |               |   \   /   |    \  |   |   \  Line |
+      |  +-------+      internal |    \ |    o-------/   o    | LTP 2|
+      |                 AD ports |     -o  6 |      \    | 2  o------o->
+      |                          |      |    o-------\---o    |   |  |
+      |                          o       \___|        |  |___/    |  |
+      |                          |                    |       DEG2|  |
+      |                          |                    |   ___     |  |
+      |                          |                    |  |   \  Line |
+      |                          |                    \--o    | LTP 3|
+      |                          |                       | 3  o------o->
+      |                          o                       o    |   |  |
+      |                          |                       |___/    |  |
+      |                          |                            DEG3|  |
+      |                          +--------------------------------+  |
+      +--------------------------------------------------------------+
+
+
+       Figure 29: Protected TTP with Restricted Connectivity between
+                ROADM Add-Drop Ports and ROADM Degree Ports
+
+   {
+     "roadm-path-impairments": [
+       {
+         "roadm-path-impairments-id": 1,
+         "roadm-add-path":
+         "Add path impairments from TTP 1 or TTP 2 to any LTPs."
```

Comment:
there are no TTP1 or TTP2 in figure 28 and 29

Reply:
@EstherLerouzic : we have not reprted in ascii figure all the TTP, only one is represented , the reader need to look at the text to understand the sceanrio of the example. In case of TTP 1 and 2 , what is represented is always full connectivity. In case of TTP3 and 4 restricted connectivities.
We will add a sentence in the text to clarify .